Makefile

```diff
@@ -12,15 +10,7 @@ form.pdf: form.tex fig-SourceBots.pdf
 	pdflatex $(PDFLATEXFLAGS) $<
 
 fig-%.pdf: fig-%.svg
-ifeq ($(COMPILESVG),inkscape)
```

Comment:
Not sure if removing the option for inkscape was necessary, but it definitely makes sense to have svg2pdf used by default.

Reply:
I disagree -- users editing SVGs are likely to have inkscape on their machines, they may well not have `svg2pdf` (something I'd not encountered before seeing it in this `Makefile`).

I'd also note that this is a breaking change, yet something easy for the CI to configure for itself. I'd rather that we didn't change the `Makefile` in this manner just to appease the CI.

Reply:
Doesn't look like `make` has a nice way of overriding variables at the command line, so time to work some magic!

Reply:
Huh, what's the point of the variables then?

https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding documents that they can be overridden on the command-line. Is that a GNU-only extension?

Reply:
Oh, damn, that seems to work on my machine, will try instead of my hack